index.ts

```diff
@@ -8,4 +8,5 @@ export * from './account';
 export * from './gaia';
 export * from './ledger';
 export * from './coins';
-export * from './seedVault';
+export { default as SeedVault } from './seedVault';
```

Comment:
This won't export the interfaces/types and the enum. I think you also need `export * from './seedvault'`

Reply:
@m-aboelenein just re-checking if you considered this